DirectEmployers/MyJobs
# [PD 3011] replace loading spinners

## What's this PR do?
This PR replaces all of the loading spinners with the one we use for analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside PRM, Reports, or Manage Users. Anywhere that may have the loading spinners for data loading

## How should this be manually tested?
Pull Down the Repository. There are a couple of the apps that utilize the loading spinners which should be Analytics, PRM, Reports, and Manage Users. Check and Make sure the loading spinner is the same blue and gray one that we use for analytics

## Any background context you want to provide?
Styling doesn't have to be the same as far as matching analytics goes. Only the spinner has to be the same. For all of the apps except for Analytics, we won't have an overlay for it and just the spinner in general
